operations/host_spawn.go

```diff
@@ -1375,6 +1375,62 @@ Examples:
 	}
 }
 
+func hostFindBy() cli.Command {
+	const (
+		ipAddressFlagName = "ip-address"
+		legacyFlagName    = "legacy"
+	)
+	return cli.Command{
+		Name:  "get-host",
+		Usage: "look up existing hosts",
```

Comment:
"hosts" isn't true so it's only one host; it also returns specifically just the link for that host, so maybe "get link to existing host"

Reply:
done